riva/clients/asr/riva_streaming_asr_client.cc

```diff
@@ -144,6 +147,7 @@ main(int argc, char** argv)
   str_usage << "           --stop_threshold=<float>" << std::endl;
   str_usage << "           --stop_threshold_eou=<float>" << std::endl;
   str_usage << "           --custom_configuration=<key:value,key:value,...>" << std::endl;
+  str_usage << "           --speaker_diarization=<true|false>" << std::endl;
```

Comment:
@pskrunner14 to push changes to enable/disable diarization based on this flag

Reply:
Done